docs/src/main/asciidoc/build-items.adoc

```diff
@@ -0,0 +1,630 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+= Quarkus - Build Items
+
+Here you can find a list of Build Items and the extension that provides them:
+
+== Agroal - Database connection pool
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/spi/src/main/java/io/quarkus/agroal/spi/JdbcDriverBuildItem.java[`io.quarkus.agroal.spi.JdbcDriverBuildItem`, window="_blank"]
```

Comment:
Can the script read the current HEAD and propose that instead of "master" 
e.g proposing 
 - "https://github.com/quarkusio/quarkus/blob/d30c121757d8524a3d5eee6ed2dfcb956a094853/extensions/agroal/spi/src/main/java/io/quarkus/agroal/spi/JdbcDriverBuildItem.java" instead of 
- "https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/spi/src/main/java/io/quarkus/agroal/spi/JdbcDriverBuildItem.java"
?

Reply:
This was we could just use the script proposed in the description of this PR to generate this file which will point to source of the latest releases (which is not master since it can contain changes that are not yet released). Moreover, we'll avoid committing this file since it can be generated. Much like what we have now for the configuration reference docs

Reply:
Hmmm, not a bad idea. Perhaps something I can provide that in a separate PR. Thanks!